src/hooks/checkLocationPermission.tsx

```diff
@@ -39,8 +39,16 @@ export const requestPermissions = () => {
       {
         text: "Open Settings",
         style: "default",
+        isPreferred: true, // @ts-ignore This is only available on iOS
```

Comment:
This doesnt seem to make a visual change on my end

Reply:
It was in the docs but I guess it doesn't work 🤣. Will remove